dali/test/python/test_operator_readers_nemo_asr.py

```diff
@@ -174,3 +175,29 @@ def test_decoded_vs_generated(batch_size=3):
       # String comparison (utf-8)
       assert text_non_ascii_str == ref_text_non_ascii_literal[idx], \
           f"'{text_non_ascii_str}' != '{ref_text_non_ascii_literal[idx]}'"
+
+
+alias_batch_size=64
```

Comment:
Nitpick: this `alias` doesn't look right here...

Reply:
done